resources/popup.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function () {
```

Comment:
I think @mdingjan means the two functions `read_status` and `set_popup`. I think it doesn't matter where these functions are, being inside the scope of the `ready` callback only means that they can only be used inside that callback, so in this case they don't pollute the global scope :)